src/main/groovy/com/bmuschko/gradle/docker/tasks/image/DockerLoadImage.groovy

```diff
@@ -6,10 +6,11 @@ import org.gradle.api.tasks.Input
 class DockerLoadImage extends AbstractDockerRemoteApiTask {
 
     @Input
-    InputStream imageStream
+    String inputFilePath
```

Comment:
Lets keep the focus of this PR purely on `DockerSaveImage` and remove this change.

Reply:
@kalap92 did we want to remove this to get the PR merged?